inc/blocks/ab-test/edit.js

```diff
@@ -212,7 +213,10 @@ const Edit = ( {
 				{ variants.map( ( variant, index ) => (
 					<VariantPanel
 						key={ `variant-settings-${ variant.clientId }` }
+						blockId= { attributes.clientId }
```

Comment:
I get where you were going with this now. It does avoid needing to make an API request to get the XB's post ID.

Reply:
Bonus space here:
```suggestion
						blockId={ attributes.clientId }
```